discussion_grading/enums.py

```diff
@@ -0,0 +1,17 @@
+"""This module contains enums used in the Discussion Grading XBlock."""
+
+from enum import Enum
+
+from discussion_grading.utils import _
+
+
+class DiscussionGradingMethod(Enum):
+    """
+    Enum for discussion grading method.
+
+    - MINIMUM_PARTICIPATIONS: Learners are graded based on the minimum number of participations
+    - AVERAGE_PARTICIPATIONS: Learners are graded based on the average number of participations
+    """
+
+    MINIMUM_PARTICIPATIONS = _("Minimum Participations")
+    AVERAGE_PARTICIPATIONS = _("Average Participations")
```

Comment:
Same suggestion here about having a separate file for enums. Should we include them in the utils.py file or in the discussion_grading.py file?

Reply:
I left it in the main module since it is not used anywhere else.
Done: https://github.com/eduNEXT/xblock-discussion-grading/commit/1ebe6648e62aedfd42acf1d736ba9db8c31c59c8